blog/2025-03-14-media-player-toggle-adjusted.md

```diff
@@ -0,0 +1,15 @@
+---
+author: Erik Montnemery
+authorURL: https://github.com/emontnemery
+title: "Changed implementation of media player toggle"
+---
+
+The implementation of the toggle action service has been adjusted in the media player base class,
+the media player will be turned on if it is in states `off` or `standby`, otherwise it will be
+turned off.
```

Comment:
```suggestion
The implementation of the toggle action service has been adjusted in the media player base class.
The media player will now be turned on if it is in states `off` or `standby`, otherwise it will be
turned off.
```